lib/api_browser/app/views/controller.html

```diff
@@ -5,7 +5,7 @@
   <div class="row">
     <div class="col-lg-12">
       <h1 class="page-header">
-        {{ controller.name | resourceName }}
+        {{ controller.display_name | resourceName }}
```

Comment:
I'd imaging something like `controller.display_name || (controller.name | resourceName)` should do the trick.